app/controllers/order_items_controller.rb

```diff
@@ -1,25 +1,43 @@
 class OrderItemsController < ApplicationController
 
+    def new
```

Comment:
I think you're missing tests for this controller, add tests and we can wrap this up